matsim/runtime/eqasim.py

```diff
@@ -11,7 +11,7 @@ def configure(context):
     context.stage("matsim.runtime.maven")
     context.config("eqasim_java_package")
 
-    context.config("eqasim_version", "1.2.1")
+    context.config("eqasim_version", "1.5.0")
```

Comment:
The old MatSim repo `dl.bintray.com` no longer exists:
https://github.com/eqasim-org/eqasim-java/blob/v1.2.1/pom.xml

However, v1.5.0 eqasim uses `repo.matsim.org` and still works:
https://github.com/eqasim-org/eqasim-java/blob/v1.5.0/pom.xml